src/backend/dist.js

```diff
@@ -26,14 +26,14 @@ const downloadFile = function(url, file_path, next) {
   });
 
   fileStream.on('finish', function() {
+    next();
```

Comment:
You might also want to call `next()` inside the `catch` below, if you want the process to continue after an error, instead of stalling.

Alternatively you could call `next(err)` from the catch, if you want to pass the error back for the caller to handle, in which case you shouldn't really need to log here.